src/fights.ts

```diff
@@ -290,66 +303,102 @@ const freeFightSources = [
     }
   ),
 
+  //Initial 9 Pygmy fights
   new FreeFight(
     () =>
-      get("questL11Worship") !== "unstarted" ? clamp(10 - get("_drunkPygmyBanishes"), 0, 10) : 0,
+      get("questL11Worship") !== "unstarted" ? clamp(9 - get("_drunkPygmyBanishes"), 0, 9) : 0,
     () => {
       putCloset(itemAmount($item`bowling ball`), $item`bowling ball`);
-      retrieveItem(10 - get("_drunkPygmyBanishes"), $item`Bowl of Scorpions`);
+      retrieveItem($item`Bowl of Scorpions`);
```

Comment:
Should this be retrieveItem(9, $item`Bowl of Scorpions`)? Or is this sequence run once for every pygmy?

Reply:
Runs once per pygmy, but with that said it should probably retreive 9-banishes so it only performs the retrieval once

Reply:
Resolved as of ef336bb